scripts/PearlPowderISIS/pearl_routines.py

```diff
@@ -0,0 +1,1335 @@
+# pylint: disable=anomalous-backslash-in-string, global-variable-undefined, global-variable-not-assigned
```

Comment:
I'd add a comment about global variables, why we are leaving them here for now. It's a necessary evil but it is evil.